### ISSUE TITLE
Finalization of vm.yml

### ISSUE THREAD

**Issue by mblaschke** [mblaschke/vagrant-development#62]
Remove configuration options from `Vagrantfile` and move all to `vm.yml`

**Comment by elmar-hinz** [comment]
Would it be possible to add customizable mounts in this step? 

In example I need to mount: /Volumes/Work -> /Volumes/Work by NFS.

Maybe generally mount /Volumes/ -> /Volumes/  on OS X? But that isn't that easy.

This does work:

```
config.vm.synced_folder "/Volumes/Work/", "/Volumes/Work/", :nfs => { :mount_options => [ "dmode=775", "fmode=774" ] }
```

This doesn't:

```
config.vm.synced_folder "/Volumes/", "/Volumes/", :nfs => { :mount_options => [ "dmode=775", "fmode=774" ] }
```

There is automounting the the way.

**Comment by mblaschke** [comment]
There will be a array with shared folders:
https://github.com/webdevops/vagrant-development/blob/feature/refactor-vagrantfile/vm.yml#L79

**Comment by elmar-hinz** [comment]
Awesome!

**Comment by mblaschke** [comment]
Fixed with f3636213da4ec2daa9c0cad4f826689713101f4f
